android/app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,9 @@
      <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
      <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM" />
       <uses-permission android:name="android.permission.POST_NOTIFICATIONS"/>
+        <uses-permission android:name="android.permission.READ_MEDIA_AUDIO" />
+  <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
+  <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The permissions `READ_MEDIA_AUDIO`, `READ_MEDIA_IMAGES`, and `READ_MEDIA_VIDEO` are not standard Android permissions. If these are custom permissions, ensure they are properly defined in your app or the library you're using. If they are not needed, consider removing them to respect user privacy and minimize the app's permission footprint.
```diff
- <uses-permission android:name="android.permission.READ_MEDIA_AUDIO" />
- <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
- <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->